src/routes/root.rs

```diff
@@ -8,19 +8,26 @@ use super::AppState;
 
 pub async fn root<S: AppState>(
     State(state): State<S>,
-    Extension(cookie): CookieUserState,
+    Extension(user_state): CookieUserState,
 ) -> Markup {
-    let search_results = if let Some(user_state) = cookie {
```

Comment:
thanks to my change in middlewares.rs, we no longer need to do this error-handling logic to get the user-state. we can just blindly accept that it's valid because we trust the `parse_cookie` function in middlewares.rs to do that for us